libs/domain/picking/src/components/picking-product-card/picking-product-card.component.ts

```diff
@@ -0,0 +1,262 @@
+import { asyncState, i18n, valueType } from '@spryker-oryx/utilities';
+import { html, LitElement, TemplateResult } from 'lit';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { property, state } from 'lit/decorators.js';
+import { when } from 'lit/directives/when.js';
+import { BehaviorSubject } from 'rxjs';
+import {
+  ItemsFilters,
+  PickingListItem,
+  ProductItemPickedEvent,
+  SummaryInfo,
+} from '../../models';
+import { styles } from './picking-product-card.styles';
+
+export const EVENT_CHANGE_NUMBER_OF_PICKED = 'oryx.change-number-of-picked';
+export const EVENT_SUBMIT = 'oryx.submit';
+export const EVENT_EDIT = 'oryx.edit';
+
+export class PickingProductCardComponent extends LitElement {
+  static styles = styles;
+
+  @property() productItem?: PickingListItem;
```

Comment:
What are the benefits of having an interface for just this two properties?

Reply:
This is public interface for the element. This allows us:
1) To get available properties and their types when query the element
2) Use this interface for storybook as a type for storybook properties

But you might be right. If there are no stories for this component, this can be useless. But if it requires stories - better to create interface and reuse it. Up to you

Reply:
Let's change/adapt this if we need stories for that in another sub-task or independent task